src/test/resources/mockdata/invoiceLines/invoice_lines.json

```diff
@@ -0,0 +1,125 @@
+{
+  "invoice_lines": [
```

Comment:
According to [invoice_line_collection.json](https://github.com/folio-org/acq-models/blob/master/mod-invoice-storage/schemas/invoice_line_collection.json#L6) this property is `invoiceLines`